protected/humhub/modules/content/components/ContentContainerModuleManager.php

```diff
@@ -280,4 +294,13 @@ public static function getContentContainerQueryByModule($id)
         return $query;
     }
 
+    /**
+     * Check module for "set as default" state as "Always active"
+     * @param $id
+     * @return bool
+     */
+    public function isAlwaysActive($id)
+    {
+        return self::getDefaultState($this->contentContainer->className(), $id) === self::STATE_ALWAYS_ACTIVE;
```

Comment:
`className()` is deprecated, maybe use `get_class()` instead

Reply:
Considering that most users tend to use newer versions of PHP it would be for the best to do this

Reply:
Done! Thank you for the useful comment!